packages/expo-font/ios/FontExceptions.swift

```diff
@@ -12,6 +12,12 @@ internal final class FontCreationFailedException: GenericException<String> {
   }
 }
 
+internal final class FontNoPostScriptException: GenericException<String> {
+  override var reason: String {
+    "Could not create font from loaded data for '\(param)'. Could not obtain font's 'postScriptName'."
```

Comment:
Why quotes for `postScriptName`? Also, we often try to make error messages so that they are descriptive but in one sentence (like changelog entries) and without the period at the end (Web and Node.js follow that rule).

```suggestion
    "Could not create font '\(param)' from loaded data because it is missing the PostScript name"
```